docs/overview.rst

```diff
@@ -52,12 +52,11 @@ Here's a bunch of top-level directories and what's in them::
     docs/                   -- documentation
     favicons/               -- favicons used by Symbols webapp
     frontend/               -- Symbols webapp JavaScript frontend
-    schemas/                -- API schemas
```

Comment:
For my notes: The schemas held the symbolication v4 and v5 reponse jsonschema specs. When we built Eliot and then moved that to a new repo, we didn't have anymore schemas.